pkg/actions/addon/get.go

```diff
@@ -99,6 +99,10 @@ func (a *Manager) findNewerVersions(addon *api.Addon) (string, error) {
 		return "", err
 	}
 
+	if len(versions.Addons) == 0 {
+		return "-", nil
+	}
+
```

Comment:
fixes a panic in the integration test, think this is the first time we've had no new versions available since addons got released.